src/services/User/getAllUsers.ts

```diff
@@ -0,0 +1,7 @@
+import { TUser } from '../../models/User';
+import { USERS_DB } from '../../store';
+
+const getAllUsers = (): Array<TUser> => USERS_DB.filter(({ isDeleted }: TUser) => !isDeleted);
```

Comment:
not need explicitly specify `TUser`
`const getAllUsers = (): Array<TUser> => USERS_DB.filter(({ isDeleted }) => !isDeleted);`